EstateManagement/Startup.cs

```diff
@@ -203,9 +220,31 @@
             });
             services.AddSingleton<HttpClient>();
         }
-
+        
+        /// <summary>
+        /// Configures the middleware services.
+        /// </summary>
+        /// <param name="services">The services.</param>
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("HealthCheck"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Read Model Server",
+                                  failureStatus:HealthStatus.Degraded,
+                                  tags:new string[] {"db", "sql", "sqlserver"})
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=6232863)